.github/CODEOWNERS

```diff
@@ -122,10 +122,11 @@ packages/react-card/ @microsoft/cxe-red @khmakoto
 packages/react-checkbox/ @microsoft/cxe-red @khmakoto
 packages/react-components/ @layershifter @miroslavstastny
 packages/react-focus/ @microsoft/cxe-red @khmakoto
-packages/react-image/ @cxe-prg
+packages/react-image/ @microsoft/@cxe-prg
```

Comment:
It seems that this PR breaks `CODEOWNERS` at all... @Hotell Can you please check?

![image](https://user-images.githubusercontent.com/14183168/126167100-0d8e69c6-863e-444a-bffe-a04fa634eb48.png)
_Even Github highlighter considers these entries invalid_

---

There should be a single `@`:

```diff
-packages/react-image/ @microsoft/@cxe-prg
+packages/react-image/ @microsoft/cxe-prg
```

Reply:
well, that's why I hate search/replace without proper context awareness :D , thx